### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by Darnele91**
Fixed all the broken links. 
I hope this looks good. I had some trouble with git.

### CODE REVIEW

File: docs-gatsby/src/pages/kafka/arch.mdx [hunk 1]
```diff
@@ -118,7 +118,7 @@ Zookeeper is not CPU intensive and each server should have a least 2 GB of heap
 
 ## Kubernetes Operator
 
-It is important to note that the deployment and management of stateful application in Kubernetes should, now, use the proposed [Operator Framework](https://github.com/operator-framework) introduced by Red Hat and Google. One important contribution is the [Strimzi Kafka operator](https://github.com/strimzi/strimzi-kafka-operator) that simplifies the deployment of Kafka within k8s by adding a set of operators to deploy and manage Kafka clusters, topics, users and more. Here is [our study](../deployments/strimzi/deploy.md) about Strimzi deployment on Openshift cluster.
+It is important to note that the deployment and management of stateful application in Kubernetes should, now, use the proposed [Operator Framework](https://github.com/operator-framework) introduced by Red Hat and Google. One important contribution is the [Strimzi Kafka operator](https://github.com/strimzi/strimzi-kafka-operator) that simplifies the deployment of Kafka within k8s by adding a set of operators to deploy and manage Kafka clusters, topics, users and more. Here is [our study](../deployments/strimzi/deploy/) about Strimzi deployment on Openshift cluster.
```
Comment (osowski) [inline docs-gatsby/src/pages/kafka/arch.mdx:121]: The `our study` link should point to `/deployments/strimzi/deploy/`

File: docs-gatsby/src/pages/kafka/mirrormaker.mdx [hunk 2]
```diff
@@ -219,7 +219,7 @@ What needs to be done:
 
 * Get a OpenShift cluster in the same data center as Event Streams service: See this [product introduction](https://cloud.ibm.com/kubernetes/catalog/about?platformType=openshift).
 * Create a project in OpenShift, for example: `mirror-maker-2-to-es`. Remember it is mapped to a namespace in Kubernetes.
-* At the minimum, to run Mirror Maker 2, we need to deploy the Strimzi Custom Resource Definitions, and the Mirror Maker 2.0 operator. See the detail in sections from the [deployment note](strimzi-deploy.md). The 0.17.0 source is in [this repository](https://github.com/strimzi/strimzi-kafka-operator/releases), unzip and use the `install` folder with Strimzi installation instructions.
+* At the minimum, to run Mirror Maker 2, we need to deploy the Strimzi Custom Resource Definitions, and the Mirror Maker 2.0 operator. See the detail in sections from the [deployment note](strimzi-deploy/). The 0.17.0 source is in [this repository](https://github.com/strimzi/strimzi-kafka-operator/releases), unzip and use the `install` folder with Strimzi installation instructions.
```
Comment (osowski) [inline docs-gatsby/src/pages/kafka/mirrormaker.mdx:222]: Link for `[deployment note]` should point to `deployments/strimzi/deploy/`

File: docs-gatsby/src/pages/methodology/eventstorming.mdx [hunk 3]
```diff
@@ -50,7 +50,7 @@ A timeline of domain events is the critical output of the first step in the even
 One-View Figure.
 
 * **Actors** consume data by using a user interface and use the UI to interact with the system via commands. Actors could also be replace by articial intelligent agents.
-* **Commands** are the result of some user decision or policy, and act on relevant data which are part of a Read model in the [CQRS](../design-patterns/cqrs.md) pattern.
+* **Commands** are the result of some user decision or policy, and act on relevant data which are part of a Read model in the [CQRS](../design-patterns/cqrs/) pattern.
```
Comment (osowski) [inline docs-gatsby/src/pages/methodology/eventstorming.mdx:53]: This should point to `/design-patterns/cqrs`

File: docs-gatsby/src/pages/methodology/eventstorming.mdx [hunk 4]
```diff
@@ -95,7 +95,7 @@ The second type of boundary is a subject boundary. You can detect a subject boun
 
 You can delineate these different sets of simultaneous event streams by applying blue painter’s tape horizontally, dividing the board into different swim lanes.
 
-Below is an example of a set of ordered domain events with pivotal events and subject swim lanes indicated.  This example comes from applying event storming to the domain of container shipping process and is discussed in more detail in the [container shipment analysis example](https://github.com/ibm-cloud-architecture/refarch-kc/blob/master/analysis/readme.md). When the reefer container is plugged to the Vessel, it starts to emit telemetries, we change context.
+Below is an example of a set of ordered domain events with pivotal events and subject swim lanes indicated.  This example comes from applying event storming to the domain of container shipping process and is discussed in more detail in the [container shipment analysis example](https://github.com/ibm-cloud-architecture/refarch-kc/blob/master/analysis/readme/). When the reefer container is plugged to the Vessel, it starts to emit telemetries, we change context.
```
Comment (osowski) [inline docs-gatsby/src/pages/methodology/eventstorming.mdx:98]: This should point to https://ibm-cloud-architecture.github.io/refarch-kc/analysis/readme

File: docs-gatsby/src/pages/methodology/eventstorming.mdx [hunk 5]
```diff
@@ -120,7 +120,7 @@ It is possible to chain events and commands as presented in the "one view" figur
...
+You can't truly define a command without understanding the data that is needed for the command to execute in order to produce the event. You can identify several types of data during this step.  First, users (personas) need data from the user interface in order to make decisions before executing a command.  That data forms part of the read model in a [CQRS](../design-patterns/cqrs/) implementation. For each command and event pair, you add a data description of the expected attributes and data elements needed to take such a decision. Here is a simple example for a `shipment order placed` event created from a `place a shipment order action`.
```
Comment (osowski) [inline docs-gatsby/src/pages/methodology/eventstorming.mdx:123]: CQRS should point to `/design-patterns/cqrs`

File: docs-gatsby/src/pages/methodology/eventstorming.mdx [hunk 6]
```diff
@@ -232,4 +232,4 @@ The K Container Shipment use case demonstrates an implementation solution to val
 * [Domain drive design with event storming introduction video](https://learning.oreilly.com/videos/domain-driven-design-distilled/9780134593449/9780134593449-DDDD_07_01?autoplay=false)
 * [Patterns related to Domain Driven Design](https://martinfowler.com/tags/domain%20driven%20design.html) by Martin Fowler
 * [Kyle Brown - IBM - Apply Domain-Driven Design to microservices architecture](https://www.ibm.com/cloud/garage/practices/code/domain-driven-design/overview)
-* [Applying DDD and event storming for event-driven microservice implementation from our own work](./ddd/)
+* [Applying DDD and event storming for event-driven microservice implementation from our own work](ddd/)
```
Comment (osowski) [inline docs-gatsby/src/pages/methodology/eventstorming.mdx:235]: This should point to `/methodology/ddd`

File: docs-gatsby/src/pages/kafka/mirrormaker.mdx (hunk 2 above)
Comment (jbcodeforce) [reply docs-gatsby/src/pages/kafka/mirrormaker.mdx:222]: This one will be completly updated when our work on a separate repo will be completed

**Pull request merged**